doc/examples/segmentation/plot_mask_slic.py

```diff
@@ -42,7 +42,7 @@
 mask = morphology.opening(mask, morphology.disk(3))
 
 # SLIC result
-slic = segmentation.slic(img, n_segments=200)
+slic = segmentation.slic(img, n_segments=200, start_label=1)
```

Comment:
I guess start_label=1 is done to be consistent with the masked case, so that only unlabeled areas would have a label of 0?

Reply:
Yes, it is also related to the issue #3722 ;)